test/code_gen/test_code_generate.mlir

```diff
@@ -0,0 +1,35 @@
+// RUN: mlir-neura-opt %s \
+// RUN:   --assign-accelerator \
+// RUN:   --lower-llvm-to-neura \
+// RUN:   --leverage-predicated-value \
+// RUN:   --transform-ctrl-to-data-flow \
+// RUN:   --insert-data-mov \
+// RUN:   --map-to-accelerator="mapping-strategy=heuristic" \
+// RUN:   --generate-code
+// RUN: FileCheck %s --input-file=generated-instructions.yaml --check-prefix=CHECK
+// RUN: FileCheck %s --input-file=generated-instructions.asm --check-prefix=ASM
+
+// CHECK-DAG: "name": "arith.constant"
+// CHECK-DAG: "name": "neura.data_mov"
+// CHECK-DAG: "name": "neura.fadd"
+// CHECK-DAG: "neura.return"
+// CHECK-DAG: "CompiledII": 1
+// CHECK-DAG: "ResMII": 1
+// CHECK-DAG: "dst_direction": "East"
+// CHECK-DAG: "src_direction": "Local"
+// CHECK-DAG: "tile_instructions"
+// CHECK-DAG: "func_name": "test"
+
+// ASM: PE(0,2):
```

Comment:
Plz provide the entire ASM for this test, and CHECK ASM for all lines, so we get better idea about what we are generating.